vendor/github.com/google/cel-go/parser/parser.go

```diff
@@ -46,7 +46,7 @@ func NewParser(opts ...Option) (*Parser, error) {
 		}
 	}
 	if p.maxRecursionDepth == 0 {
-		p.maxRecursionDepth = 200
+		p.maxRecursionDepth = 250
```

Comment:
this was an increase in the max recursion limit that can be parsed? would this effectively relax validation and accept expressions that would previously have been rejected?

Reply:
The max recursion depth usually comes into play for expressions like `a && b && c && d && .....`, but there's a more detailed write-up of what's supported in the cel-spec just below the syntax: https://github.com/google/cel-spec/blob/master/doc/langdef.md#syntax

Reply:
I have added a test for cel maxRecursionDepth in the lates commit. Seems like the setting is not exactly mentioned here(250). Please feel free to let me know if I missed anything. Thank you

Reply:
The 250 limit is an approximation of when recursion during parsing leads to long parse times. In terms of the documented support for recursion, the number of consecutive operators can result in loops back through the `expr` parse node that approach the 250 number. The number was determined and tuned as a result of fuzzing tests which pointed out problematic cases. Think of it more like a fail-safe against malicious inputs.